doc/source/config.rst

```diff
@@ -0,0 +1,130 @@
+Configuration
+=============
+
+Satpy has two levels of configuration that allow to control how Satpy and
+its various components behave. There are a series of "settings" that change
+the global Satpy behavior. There are also a series of "component
+configuration" YAML files for controlling the complex functionality in readers,
+compositors, writers, and other Satpy components that can't be controlled
+with traditional keyword arguments.
+
+Settings
+--------
+
+There are configuration parameters in Satpy that are not specific to one
+component and control more global behavior of Satpy. These parameters can be
+set in one of three ways:
+
+1. Environment variable
+2. YAML file
+3. At runtime with ``satpy.config``
+
+This functionality is provided by the :doc:`donfig <donfig:configuration>`
+library. The currently available settings are described below.
+Each option is available from all three methods. If specified as an
+environment variable or specified in the YAML file on disk, it must be set
+**before** Satpy is imported.
+
+**YAML Configuration**
+
+YAML files that include these parameters can be in any of the following
+locations:
+
+1. ``<python environment prefix>/etc/satpy/satpy.yaml``
+2. ``~/.config/satpy/satpy.yaml``
+3. ``~/.satpy/satpy.yaml``
+4. ``<SATPY_CONFIG_PATH>/satpy.yaml`` (see :ref:`config_path_setting` below)
```

Comment:
How about non-linux platforms?

Reply:
This is using an environment variable which should be fine in Windows and the other ones could still exist. I'm not sure what `~` maps to on Windows (probably the Users directory), but I think the understanding is that Python will handle it. A lot of these paths (1, 2, and 3) are taken from the dask defaults that were copied to donfig. I'm ok not including them, but I don't see why they can't be included.

Reply:
I'm not pushing for non-inclusion, rather provide the same list for other platforms...

Reply:
I think the list is the same for all platforms unless I use `user_config_dir` from appdirs as you suggested in your other comment.